docs/add-ins/addin-tutorial.md

```diff
@@ -973,20 +1079,20 @@ Now that the UI is implemented, let's add the code behind it. Create a file name
                 Office.context.mailbox.item.body.setSelectedDataAsync(content,
                   {coercionType: Office.CoercionType.Html}, function(result) {
                     if (result.status == 'failed') {
-                      showError('Could not insert Gist: ' + result.error.message);
+                      showError('Could not insert gist: ' + result.error.message);
                     }
                 });
               } else {
                 showError('Could not create insertable content: ' + error);
               }
             });
           } else {
-            showError('Could not retrieve Gist: ' + error);
+            showError('Could not retrieve gist: ' + error);
           }
         });
       });
 
-      // When the settings icon is clicked, open the settings dialog
+      // When the settings icon is selected, open the settings dialog
```

Comment:
"Select the settings icon to open the settings dialog"

Reply:
I've changed "clicked" to "selected" but left the phrasing the same -- since these are code comments describing what will happen when the button is selected (not instructions for the reader).